### ISSUE TITLE
Redstone tool crash

### ISSUE THREAD

**Issue by unknowndobo** [lumien231/Random-Things#286]
Hi Lumien, I've been having great fun with your mod on Skyfactory 3 lately but i'm now experiaencing a weird error.  I've set my factory up to use your redstone interfaces as a way to send signals wheni have too many of an item in my refined storage, the signal they send bounces to whatever it is that's creating/farming/sieving the specific item and redirects it to somewhere more useful, like a loot recycler. 
I've set up a couple of these systems,(redstone essence becomes redstone, when that's full it creates redstone blocks, when that's full it recycles) but now every time i select the redstone tool in my hotbar the game crashes. I get a static screen similar to if slenderman had popped into my world and then i'm back at my desktop.  
I've tested with other items in the mod pack (redstone interfaces have no issue, neither does the magic hood) , and some of the other mod packs but so far i've only been able to replicate it with the redstone tool, and it is every time i select the redstone tool.  It doesn't matter whether i scroll to it with the mouse, press the number its associated to or place it directly from my inventory into my hand, game crashed. 
I've been trying for the last couple of hours to get a crash log but it seems that minecraft hasn't been creating one, the console seems to stop tracking as soon as it happens as well. I'm looking into another way of getting a log file to you (playing around with text bat files) but i've long since passed into realms where i don't know what i'm doing. 
Have you come across this error before and is there a way to fix it?
Many thanks
Dobo

**Comment by lumien231** [comment]
Well without a crash log i can't really do much, have you looked through the log files in your logs folder?

Otherwise can you reproduce this on a fresh world?

**Comment by unknowndobo** [comment]
Yeah i've looked through the log files but there doesn't seem to be anything useful in them, it just has the long blurb as it's opening minecraft and then nothing when there are glitches occuring. 
I tried a couple of other things since my last message. I had a theory that the red lines that come out of the interfaces might be causing the problem so i moved away to a different chunk where i couldn't see the interfaces at all and tried switching to the redstone tool, no problem, no graphics glitches no crashes.  I flew back to my factory and let the chunk load but made sure i couldn't see anywhere that the red lines would appear then switched to the redstone tool, again no problem.  As soon as i turned around a corner to look at an allocated interface with the tool in my hand i got the crash. 
I went into twitch and allocated more memory to minecraft to see if it was a case that the red lines were eating my ram.
I set up a new skyfactory world and was able to partially reproduce it, no crash this time but i am getting the exact same graphical issues, fortunately because it didnt crash i was able to screen capture a couple of times to show you what i mean.
![minecraft 1 10 2 04_02_2018 21_30_25](https://user-images.githubusercontent.com/35978566/35782920-abd705c8-09f7-11e8-9cca-ab0508d58ad8.png)
![minecraft 1 10 2 04_02_2018 21_30_39](https://user-images.githubusercontent.com/35978566/35782921-abfbe69a-09f7-11e8-9226-ae041308cf76.png)
![minecraft 1 10 2 04_02_2018 21_27_46](https://user-images.githubusercontent.com/35978566/35782922-ac171082-09f7-11e8-8a20-8698268cf0c0.png)


Then because i was like to live on the wild side ....and i had nothing better to do, i tried seeing if i could open my origional world and reproduce the glitch without crashing.  Low, i was able to get the graphical error without the crash if i kept my distance and observed the red lines (we'll call them death rays from now on) from afar.
My world when holding a redstone tool.

![minecraft 1 10 2 04_02_2018 21_44_06](https://user-images.githubusercontent.com/35978566/35782975-6faac3ae-09f8-11e8-8da6-3c4e545a9d22.png)

My world normally
![minecraft 1 10 2 04_02_2018 21_45_28](https://user-images.githubusercontent.com/35978566/35782980-77f7160c-09f8-11e8-8774-086490fbcb43.png)

So it seems like my problem is just caused by whatever graphical effect the death rays are using and if i can keep from looking at them i don't have a problem, fortunately in my meanderings somone pointed out that redstone interfaces can be programmed using the open computer system in skyfactory which means i can avoid seeing the deathrays from now on. So as far as my specific issue goes i now have a work around, sorry if it doesn't help you get to the root of what caused it :(
